cypress/support/common.js

```diff
@@ -15,6 +15,7 @@ export const ACCOUNT_STAGE = {
   id: '23d9209a-34be-4648-8ef0-5b522f9976be',
   slug: 'testing',
   domain: 'stashify.cloud',
+  hasMultipleZones: true,
```

Comment:
why didn't I think about that trick!?!
basically, the main reason why we still have a fake zone in stage was to have the e2e tests passing… Not sure if we want to keep it or not (the fake zone isn't working, but can still be useful for UI testing purpose).